x-pack/plugins/apm/e2e/yarn.lock

```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Why do we have that `yarn.lock` file here? We are now in a single top level package.json so we should not have more than one `yarn.lock` file 😃